mteb/tasks/Classification/mya/MyanmarNews.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class MyanmarNews(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name = "MyanmarNews", 
+        dataset= {
+            "path" : "myanmar_news",
+            "revision" : "b899ec06227db3679b0fe3c4188a6b48cc0b65eb"
+        },
+        description="The Myanmar News dataset on Hugging Face contains news articles in Burmese. It is designed for tasks such as text classification, sentiment analysis, and language modeling. The dataset includes a variety of news topics in 4 categorie, providing a rich resource for natural language processing applications involving Burmese which is a low resource language.",
+        reference = "https://huggingface.co/datasets/myanmar_news",
+        type = "Classification",
+        category= "s2s",
```

Comment:
Looks more like p2p to me. These are the full articles, right? Not just single sentences

Reply:
Yess, that makes more sense. I had seen a partial view of the records. 
I have made the change locally and will push it.

Reply:
Awesome!